consoles/blackbox-overview.html

```diff
@@ -0,0 +1,54 @@
+{{ template "head" . }}
+
+{{ template "prom_right_table_head" }}
+{{ $job := .Params.job }}
+<tr>
+  <th>{{ $job }}</th>
+  <th>{{  template "prom_query_drilldown" (args (printf "sum(up{job='%s'})" $job))  }} / {{ template "prom_query_drilldown" (args (printf "count(up{job='%s'})" $job)) }}</th>
+</tr>
+
+<tr>
+  <th colspan=2>Currently</th>
+</tr>
+{{ range query (printf "probe_success{job='%s'}" $job) | sortByLabel "instance" }}
+<tr>
+  <td {{ if eq (. | value) 1.0 }}{{ else }} class="alert-danger"{{ end }}>
```

Comment:
it might be easier to put the class on the tr, though on the node console we only have it on the Up/Down td.